tests/test_csc.py

```diff
@@ -74,15 +79,28 @@ async def test_telemetry(self):
                 m_ldco48_temperature=19,
                 m_ldco48_temperature_2=19,
             )
+            self.csc.simulator.device.do_set_m_cpu800_18_power("FAULT")
+            await self.assert_next_sample(
+                topic=self.remote.evt_summaryState,
+                summaryState=salobj.State.FAULT,
+                flush=True,
```

Comment:
I am quite worried that `flush=True` adds a race condition. I think you should already have read the most recent summary state in check_standard_state_transitions (or, at worst, just be missing one. in which case you could test for it just above line 82), and if so, you could use flush=False (which is the default, so you could also omit it).